packages/ffe-buttons-react/src/BaseButton.js

```diff
@@ -39,17 +39,21 @@ const BaseButton = props => {
             ref={innerRef}
             {...rest}
         >
-            <span className="ffe-button__label">
-                {leftIcon &&
-                    React.cloneElement(leftIcon, {
-                        className: 'ffe-button__icon ffe-button__icon--left',
-                    })}
-                {children}
-                {rightIcon &&
-                    React.cloneElement(rightIcon, {
-                        className: 'ffe-button__icon ffe-button__icon--right',
-                    })}
-            </span>
+            <div className="ffe-button__label-container">
```

Comment:
I am inherently sceptical of this move of adding a container-div inside of the actual button, in order to fix a fixed, non-critical bug in a deprecated version of iOS. 

I mean, I'm sure it fixes the issue, but adding another element to the DOM like this, in one of our most widely reused components, can be prone to regressions. Also - the fewer divs we render, the better it is IMO.

Reply:
This fix isn't critical for me either and I can remove it.

http://gs.statcounter.com/ios-version-market-share/mobile/norway/#monthly-201706-201806 says  >7% anyway.